.github/demo-dashboard/Dockerfile

```diff
@@ -1,27 +1,32 @@
-ARG BASE_IMAGE
-FROM $BASE_IMAGE
+FROM ubuntu:24.04
 ARG BRANCH=main
 
 ### Build dashboard
 
+RUN apt update && apt install -y curl
```

Comment:
`python3-venv` is also required, since the `api-server` will be built during the `pnpm install` step

Reply:
Thanks for helping to test! Should be fixed with https://github.com/open-rmf/rmf-web/pull/1007/commits/e05547f9cc3d85cbdb7e500aa0ba0a0f7566076a.